### PR TITLE
Adjust code for tox

### PR DESCRIPTION
**Pull request by zbohm**
Fix code for isort, flake8, pydocstyle, mypy. Fix migrations.

Add bumpversion.cfg, CANGELOG.md, MANIFEST.in, Makefie, tox.ini.

Remove compiled trnaslations .mo files.

Rewrite setup.py into setup.cfg.

Update README.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,12 +1,4 @@
...
 *.pyo
-*.db
-.DS_Store
-.coverage
-.idea/
-local_settings.py
-*.egg-info
-build/
-dist/
-forms_builder/example_project/static/
```
Comment (tdivis) [inline .gitignore]: Why to unignore .coverage *.egg-info build dist, that's created when you just run python setup.py build/sdist or running tests with coverage.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+- Adjust code for isort, flake8, mypy, pydocstyle.
+- Optimize code for Django > 3.2.
+- Rewrite setup.py. Move version into setup.cfg.
+- Move folder example_project outside from module forms_builder and rename to name example.
+- Add tox.ini with test more python and django versions.
+
+### Removed
+
+- Support of Python < 3.9.
+- Support of Django < 3.
```
Comment (tdivis) [inline CHANGELOG.md]: I thought that it is <3.2 now?

File: README.rst [hunk 3]
```diff
@@ -225,7 +223,7 @@ in their code.
 Email Templates
 ===============
 
-The `django-email-extras`_ package is used to send multipart email
+The function ``send_mail_template`` is used to send multipart email
```
Comment (tdivis) [inline README.rst]: What is the ``send_mail_template``? Is it a Python function inside this project? If so I guess you don't have to mention it as it was here probably for considering installin the optional dependency `django-email-extras`

File: .gitignore (hunk 1 above)
Comment (zbohm) [reply .gitignore]: `*.egg-info` returned back to ignore.

File: CHANGELOG.md (hunk 2 above)
Comment (zbohm) [reply CHANGELOG.md]: Adjusted to `Support of Django < 3.1`.

File: README.rst (hunk 3 above)
Comment (zbohm) [reply README.rst]: Removed from README.

File: CHANGELOG.md (hunk 2 above)
Comment (tdivis) [reply CHANGELOG.md]: In setup.cfg, there is `django >= 3.2, < 5.0.0`, so this should be `< 3.2`

File: .gitignore (hunk 1 above)
Comment (tdivis) [reply .gitignore]: And what about `build`, `dist` and `.coverage`?

Comment (zbohm) [reply .gitignore]: These items have also been returned.

File: CHANGELOG.md (hunk 2 above)
Comment (zbohm) [reply CHANGELOG.md]: Fixed to `< 3.2`.

**Review by tdivis — APPROVED** [review]

**Pull request closed without merging**